core/src/main/java/com/microsoft/applicationinsights/TelemetryConfiguration.java

```diff
@@ -186,9 +186,6 @@ public String getInstrumentationKey() {
      * @throws IllegalArgumentException when the new value is null or empty
      */
     public void setInstrumentationKey(String key) {
-        if (!Sanitizer.isUUID(key)) {
```

Comment:
Why don't we need to validate that IKey is GUID anymore? Is it new approach across all SDKs or this removal was unintentional?

Reply:
@Dmitry-Matveev this was not unintentional. When I talked with Sergey he told that me about this. I don't remember the exact reason but he told we use some different format too.

Reply:
@Dmitry-Matveev I have removed unused imports from most of the places, removed obsolete and replaced with Depricated at maximum locations and added couple of test cases which logically cover maximum scenarios. Please review.